code/game/objects/structures/tables_racks.dm

```diff
@@ -392,6 +399,14 @@
 	return 1
 
 
+/obj/structure/table/water_act(volume, temperature, source, method)
```

Comment:
Might want to also adopt `cleaning_act` for these, so people can easily clean em with soap. I can understand how that might be a pain in the ass, though. 

You might be able to get away with applying `water_act` under `cleaning_act`?